src/elements/content-picker/Footer.tsx

```diff
@@ -93,17 +86,16 @@ const Footer = ({
                 ) : (
                     <ButtonGroup className="bcp-footer-actions">
                         <Tooltip text={cancelButtonLabel || cancelMessage}>
-                            <Button aria-label={cancelMessage} onClick={onCancel} type="button">
+                            <Button aria-label={cancelMessage} onClick={onCancel} type={ButtonType.BUTTON}>
                                 <IconClose height={16} width={16} />
                             </Button>
                         </Tooltip>
                         <Tooltip isDisabled={isChooseButtonDisabled} text={chooseButtonLabel || chooseMessage}>
                             <PrimaryButton
                                 aria-label={chooseMessage}
-                                disabled={isChooseButtonDisabled} // sets disabled attribute
```

Comment:
[no-response]: I'm pretty sure we still need this `disabled` attribute to be passed to the native HTML button for accessibility